app/gw_work.rb

```diff
@@ -0,0 +1,20 @@
+# Generated via
```

Comment:
@kerchner Should this file be here? There's a copy in `/app/models/gw_work.rb` which makes sense, but not sure why we would want this in the `/app` root folder.